ui/packages/catalog-ui-search/src/main/webapp/component/search-form/search-form.collection.js

```diff
@@ -121,10 +121,10 @@ module.exports = Backbone.AssociatedModel.extend({
   getDoneLoading: function() {
     return this.get('doneLoading')
   },
-  checkIfOwnerOrSystem: function(template) {
+  checkIfOwner: function(template) {
     let myEmail = user.get('user').get('email')
     let templateCreator = template.creator
-    return myEmail === templateCreator || templateCreator === 'system'
+    return myEmail === templateCreator
```

Comment:
✏️ 	We can simplify this to `return user.get('user').get('email') === template.creator`